### ISSUE TITLE
Bot commands in general 

### ISSUE THREAD

**Issue by MustafaYamout** [The-Coding-Academy/tca-helper-meet#10]
I mean we have a rule that says no bot commands in general but they won't listen, maybe we should like disable all bot commands in general except for Yagpdb

**Comment by Iusedmy1stnamelol** [comment]
**NO**

**Comment by pgamerx** [comment]
> I mean we have a rule that says no bot commands in general but they won't listen, maybe we should like disable all bot commands in general except for Yagpdb

Nah, that's not cool. Many other bots like Carl, Coding bot , ~Black Sheep~ are also decent and people might like to use it in general for quick commands. Moreover, it allows us to have a backup for moderation bot
